tests/test_hvac_csc.py

```diff
@@ -131,6 +123,32 @@ async def test_standard_state_transitions(self):
     async def test_bin_script(self):
         await self.check_bin_script(name="HVAC", index=None, exe_name="run_hvac.py")
 
+    async def _retrieve_all_telemetry(self):
+        all_telemetry = {}
+        for topic in HvacTopic:
+            telemetry_topic = getattr(self.remote, "tel_" + topic.name)
+            all_telemetry[topic.name] = await telemetry_topic.next(flush=False)
+        return all_telemetry
+
+    async def _verify_telemetry(self, subsystem):
+        # Loop over all telemetry topics and verify the status
+        all_telemetry = await self._retrieve_all_telemetry()
+        for name, telemetry in all_telemetry.items():
+            topic = HvacTopic[name]
+            if topic.value in xml.TOPICS_ALWAYS_ENABLED:
+                # Skip always enabled subsystems.
+                continue
```

Comment:
Can you do anything useful to check these?

Reply:
Yes, their telemetry always is present so I should check that. As a matter of fact I intended to do that but forgot. Thanks!

Reply:
I added checks for those topics as well. It turned out to be complex since they lack the comandoEncendido and estadoFuncionamiento items so I had to add several exceptions for those.